std/uni.d

```diff
@@ -4401,7 +4405,7 @@ if (sumOfIntegerTuple!sizes == 21)
 
 }
 
-/// Type of Trie as generated by codepointTrie function.
+/// ditto
 public template CodepointTrie(T, sizes...)
```

Comment:
Ddoc definitely won't win an award in UX here:

![image](https://user-images.githubusercontent.com/4370550/34966098-7eb3eadc-fa59-11e7-912e-7d689b22207c.png)

Reply:
Hmph.  I was under the impression ddoc understood eponymous templates enough to generate sane output. Guess I was wrong. :-(

Reply:
Oh nevermind, you're talking about the two symbols differing only in uppercasing. Gah. Even I missed that. LOL...